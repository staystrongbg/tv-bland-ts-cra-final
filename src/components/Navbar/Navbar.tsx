import { Link } from 'react-router-dom';

const Navbar = () => {
  const STYLE = {
    backgroundColor: '#101010',
    color: '#f1f2f3',
    padding: '20px 60px',
  };
  return (
    <nav className='flex-between' style={STYLE}>
      <Link to='/'>
        <h1>TV Bland</h1>
      </Link>
    </nav>
  );
};

export default Navbar;
