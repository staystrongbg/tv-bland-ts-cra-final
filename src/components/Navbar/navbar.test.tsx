import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from './Navbar';

it('render app', () => {
  const render = renderer
    .create(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>
    )
    .toJSON();
  expect(render).toMatchSnapshot();
});
