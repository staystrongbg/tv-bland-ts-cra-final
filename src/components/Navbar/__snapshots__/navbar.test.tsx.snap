// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app 1`] = `
<nav
  className="flex-between"
  style={
    Object {
      "backgroundColor": "#101010",
      "color": "#f1f2f3",
      "padding": "20px 60px",
    }
  }
>
  <a
    href="/"
    onClick={[Function]}
  >
    <h1>
      TV Bland
    </h1>
  </a>
</nav>
`;
